src/giskardpy/plugin_interrupts.py

```diff
@@ -47,19 +50,64 @@ def round_js(self, js):
         """
         return tuple(round(x.position, self.wiggle_precision1) for x in js.values())
 
-
+    #@profile
     def check_fft(self):
-        traj = self.get_god_map().safe_get_data(identifier.trajectory)
+        fft_duration = 3
+        wiggle_sensitivity = 10
+        min_wiggle_frequency = 3
```

Comment:
Turn this into parameters for the config file and remove the old wiggle detection parameter.